virttest/utils_test/libvirt.py

```diff
@@ -1918,23 +1918,43 @@ def create_hostdev_xml(pci_id, boot_order=None, xmlfile=True,
     return hostdev_xml.xml
 
 
-def create_controller_xml(contr_type='scsi', contr_model='virtio-scsi',
-                          contr_index='0', contr_alias=None):
+def create_controller_xml(vm_name, contr_dict, oper="get_xml"):
```

Comment:
I have two concerning brought by  those change:
1)bind create_controller_xml with VM in earlier stage, suppose that controller may not be bind to specific VM
2)As it names imply , create is 'create' , DO NOT infuse create and get together, otherwise it will confuse users